packages/lesswrong/components/dialogues/DialoguesList.tsx

```diff
@@ -1,12 +1,138 @@
 import React from 'react';
 import { registerComponent, Components } from '../../lib/vulcan-lib';
 import withErrorBoundary from '../common/withErrorBoundary';
-import { AnalyticsContext } from '../../lib/analyticsEvents';
+import { AnalyticsContext, useTracking } from '../../lib/analyticsEvents';
 import { usePaginatedResolver } from '../hooks/usePaginatedResolver';
 import { Link } from '../../lib/reactRouterWrapper';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import { commentBodyStyles } from '../../themes/stylePiping';
+import { useUpdateCurrentUser } from "../hooks/useUpdateCurrentUser";
+import { useCurrentUser } from '../common/withUser';
+import { dialogueFacilitationMessagesABTest, useABTest } from '../../lib/abTests';
 
-const DialoguesList = () => {
+
+const styles = (theme: ThemeType): JssStyles => ({
+  dialogueFacilitationItem: {
+    paddingTop: 12,
+    paddingBottom: 12,
+    position: "relative",
+    borderRadius: theme.borderRadius.default,
+    background: theme.palette.panelBackground.default,
+    '&:hover': {
+      boxShadow: theme.palette.boxShadow.sequencesGridItemHover,
+    },
+    ...commentBodyStyles(theme),
+    lineHeight: '1.65rem',
+  },
+  content: {
+    paddingTop: 0,
+    paddingRight: 35,
+    paddingLeft: 16,
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "space-between",
+    marginRight: 75,
+    position: "relative",
+    zIndex: theme.zIndexes.spotlightItem,
+    [theme.breakpoints.down('xs')]: {
+      marginRight: 10,
+      paddingRight: 20,
+    },
+    "& p": {
+      marginBottom: 0,
+      marginTop: 5,
+    }
+  },  
+
+  closeIcon: { 
+    position: 'absolute', 
+    right: '8px',
+    top: '8px',
+    padding: '2px' 
+  }
+});
+
+const OptInComponent = ({ classes, currentUser, setShowOptIn }: { classes: ClassesType, currentUser: UsersCurrent | null, setShowOptIn: React.Dispatch<React.SetStateAction<boolean>> }) => {
```

Comment:
We don't suffix component names with `Component`, generally.  I'd also make the name more descriptive (i.e. `DialogueFacilitationPrompt` or something).

Also, if you're going to be passing in `currentUser` from the parent, it should probably not be nullable - it doesn't really make sense to show this to logged-out users.